glean-core/src/lib_unit_tests.rs

```diff
@@ -860,10 +862,20 @@ fn test_set_metrics_disabled() {
         ..Default::default()
     });
 
-    // 1. Set a configuration that disables the metric above
-    match glean.set_metrics_disabled_config("{\"metrics_disabled\": {\"category.string_metric\": true }}".to_string()) {
-        Ok(_) => log::info!("Successfuly set metrics disabled"),
-        Err(e) => log::error!("{}", e.to_string()),
+    match MetricsDisabledConfig::try_from(
+        json!(
+        {
+            "metrics_disabled": {
+                "category.string_metric": true
+            }
+        })
+        .to_string(),
```

Comment:
can we pull this out into a variable? I dislike when the `match`  header is so long